ansible.mk

```diff
@@ -0,0 +1,192 @@
+IMAGE ?= $(DEFAULT_IMAGE)
+ARCHES:="amd64" "ppc64le" "s390x" "arm64"
+
+export CGO_ENABLED:=0
+.DEFAULT_GOAL:=help
+
+.PHONY: help
+help: ## Show this help screen
+	@echo 'Usage: make <OPTIONS> ... <TARGETS>'
+	@echo ''
+	@echo 'Available targets are:'
+	@echo ''
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##############################
+# Development                #
+##############################
+
+##@ Development
+
+.PHONY: all install
+
+all: format test build/ansible-operator ## Test and Build the ansible-operator
+
+install: ## Install the ansible-operator binary
+	$(Q)$(GOARGS) go install $(GO_BUILD_ARGS) ./cmd/ansible-operator
+
+# Code management.
+.PHONY: format tidy clean cli-doc lint
+
+format: ## Format the source code
+	$(Q)go fmt $(PKGS)
+
+tidy: ## Update dependencies
+	$(Q)go mod tidy -v
+
+clean: ## Clean up the build artifacts
+	$(Q)rm -rf build
+
+lint-dev:  ## Run golangci-lint with all checks enabled (development purpose only)
+	./hack/tests/check-lint.sh dev
+
+lint-fix: ## Run golangci-lint automatically fix (development purpose only)
+	./hack/tests/check-lint.sh fix
+
+lint: ## Run golangci-lint with all checks enabled in the ci
+	./hack/tests/check-lint.sh ci
+
+setup-k8s:
+	hack/ci/setup-k8s.sh $(K8S_VERSION)
+
+##############################
+# Generate Artifacts         #
+##############################
+
+##@ Generate
+
+.PHONY: generate gen-cli-doc gen-test-framework gen-changelog
+
+generate: gen-cli-doc  ## Run all non-release generate targets
+
+gen-cli-doc: ## Generate CLI documentation
+	./hack/generate/cli-doc/gen-cli-doc.sh
+
+gen-changelog: ## Generate CHANGELOG.md and migration guide updates
+	./hack/generate/changelog/gen-changelog.sh
+
+##############################
+# Build and Release                    #
+##############################
+
+##@ Build and Release
+
+# Build and release ansible-operator.
+.PHONY: release_builds release
+
+release_builds := \
+	build/ansible-operator-$(VERSION)-aarch64-linux-gnu \
+	build/ansible-operator-$(VERSION)-x86_64-linux-gnu \
+	build/ansible-operator-$(VERSION)-x86_64-apple-darwin \
+	build/ansible-operator-$(VERSION)-ppc64le-linux-gnu \
+	build/ansible-operator-$(VERSION)-s390x-linux-gnu
+
+# TODO: add `clean` recipe as the first step
+release: $(release_builds) $(release_builds:=.asc) ## Release ansible-operator
+
+build/ansible-operator-%-aarch64-linux-gnu: GOARGS = GOOS=linux GOARCH=arm64
+build/ansible-operator-%-x86_64-linux-gnu: GOARGS = GOOS=linux GOARCH=amd64
+build/ansible-operator-%-x86_64-apple-darwin: GOARGS = GOOS=darwin GOARCH=amd64
+build/ansible-operator-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
+build/ansible-operator-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
+build/ansible-operator-%-linux-gnu: GOARGS = GOOS=linux
```

Comment:
it shows be reverting the changes made in : https://github.com/operator-framework/operator-sdk/pull/3371/files

Reply:
What is being reverted?